uint8 seq
float32 motion_base_yaw
float32[] unknown_data